### DISCUSSION TITLE
Donut Chart Implementation 🍩

### DISCUSSION THREAD

**Discussion by dibyendu-mahata** [patrykandpatrick/vico#960] [Ideas]
The donut chart is as important as the line chart, but it is missing. I kindly request you to add it as soon as possible.

**Comment by Gowsky** [comment]
Hello, @dibyendu-mahata. This is a duplicate of #426. As mentioned [here](https://github.com/patrykandpatrick/vico/discussions/426#discussioncomment-9069633), we're working on adding pie charts (which you'll also be able to style as donut charts).
